### PR TITLE
Add step to create VolumeSnapshotClass

### PR DESCRIPTION
**Pull request by duanwei33**
[test log](https://privatebin-it-iso.int.open.paas.redhat.com/?c748e00496127888#8fhtRQwNCSgMXs3UX7Db5QSKN55y5ppKUf4nFEBZg7S4)

@jhou1 @chao007 @liangxia   PTAL, thanks.

### CODE REVIEW

File: features/step_definitions/volume_snapshot.rb [hunk 1]
```diff
@@ -7,3 +7,47 @@
...
+
+  aws_file = "#{path}/volumesnapshotclass-aws.yaml"
+  gce_file = "#{path}/volumesnapshotclass-gce.yaml"
+  azure_file = "#{path}/volumesnapshotclass-azure.yaml"
+  cinder_file = "#{path}/volumesnapshotclass-cinder.yaml"
+
+  iaas_type = env.iaas[:type] rescue nil
+  case iaas_type
+  when "aws"
+    file = aws_file
```
Comment (jhou1) [inline features/step_definitions/volume_snapshot.rb:23]: I find the file names are very self-explanatory, so maybe remove the four variables and instead use:
`file = "#{path}/volumesnapshotclass-aws.yaml"` respectively.

Comment (duanwei33) [reply features/step_definitions/volume_snapshot.rb]: It makes sense, updated.

File: features/step_definitions/volume_snapshot.rb [hunk 2]
```diff
@@ -7,3 +7,42 @@
     })
   step %Q/I switch to the default user/
 end
+
+# This step is to create VolumeSnapshotClass according to different paramater on differnt platforms, also it supports custom-defined parameter
```
Comment (liangxia) [inline features/step_definitions/volume_snapshot.rb:11]: Typo of `paramater`.

Comment (duanwei33) [reply features/step_definitions/volume_snapshot.rb]: Thanks, updated.

Comment (liangxia) [reply features/step_definitions/volume_snapshot.rb]: Looks like you goes to the opposite direction

**Comment by duanwei33** [comment]
@jhou1 @liangxia Hi, please let me know if any other comments on this? Thanks a lot.

**Pull request merged**